Kwenta/kwenta
# Set favorite markets and list them at the top

## Description
Set favorite markets and list them at the top

## Related issue
#2150 
#2146
#2161

## Motivation and Context


## How Has This Been Tested?
1. Open the market page and click on the market dropdown selector
2. Click the star to set as the favorite markets and find that the favorite markets move to the top of the list
3. Go to another page and go back to the market page
4. Click on the market dropdown selector and see the favorite market are still on the top of the list
5. Click the favorite markets again to unset them and see the list is re-ordered

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4819006/230488396-2ce8d592-6943-4aee-aae8-b2b4b341f9d4.png)
